skaha/src/main/java/org/opencadc/skaha/utils/QueueUtil.java

```diff
@@ -0,0 +1,60 @@
+package org.opencadc.skaha.utils;
+
+import org.apache.log4j.Logger;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class QueueUtil {
+    private static final Logger log = Logger.getLogger(QueueUtil.class);
+
+    public static String getLocalQueue(List<String> groupNames, String jobType) throws IOException, InterruptedException, RuntimeException {
+        List<String> localQueues = new ArrayList<>();
+        for (String groupName : groupNames) {
+            String localQueue = getLocalQueueByGroupAndJobType(groupName, jobType);
+            if (localQueue != null) localQueues.add(localQueue);
+        }
+        log.debug("primary group size for the user is : " + localQueues.size());
+        if (localQueues.size() > 1) throw new RuntimeException("More than one LocalQueue for an user is Unsupported");
+        if (localQueues.isEmpty()) throw new RuntimeException("No LocalQueue available");
+        return localQueues.get(0);
+    }
+
+    private static String getLocalQueueByGroupAndJobType(String groupName, String jobType) throws IOException, InterruptedException {
+        String[] cmd = KubectlCommandBuilder.command("get")
+                .argument("-A")
```

Comment:
Listing in all namespaces (`-A`) will not be permitted in many instances, including our CANFAR deployment.  Should this only apply to the Workload Namespace (`skaha-workload`)?

Reply:
Updated with skaha workload namespace.

Reply:
Updated.